pyaccel/tracking.py

```diff
@@ -607,6 +611,11 @@ def find_orbit6(accelerator, indices=None, fixed_point_guess=None):
     """
     indices = _process_indices(accelerator, indices)
 
+    # The orbit can't be found when quantum excitation is on.
+    if accelerator.radiation_on == 2:
+        rad_stt = accelerator.radiation_on
```

Comment:
you should always save the state. Put this line outside the if statement.

Reply:
Done. I also put an observation in the docstring explaining that this function temporarily sets radiation_on to 'damping'. Thanks!